ppr-ui/src/composables/fees/enums/feeSummaryTypes.ts

```diff
@@ -3,5 +3,6 @@ export enum FeeSummaryTypes {
   DISCHARGE = 'discharge',
   NEW = 'new',
   RENEW = 'renew',
-  MHSEARCH = 'manufactured_home_search'
+  MHSEARCH = 'manufactured_home_search',
+  MHR_COMBINED_SEARCH = 'combined_home_search'
```

Comment:
@doug-lovett Same comment as above, is the api aware of these types or potentially expect something different with regards to the combined search?

Reply:
@cameron-freshworks same answer as above.

Reply:
Thank you!